includes/utilities.inc

```diff
@@ -509,8 +522,15 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
         if ($add_ellipsis) {
           $truncation_length = max($truncation_length, drupal_strlen(t('...')));
```

Comment:
the call to truncate_utf8() will add the ellipsis in the truncateText member function as its fourth parameter. This section just counts the characters to substitute from the max length. Not sure if there is a point in useing `&hellip;` here. I  could be crazy, just does not appear to work here.